src/QMCTools/convert4qmc.cpp

```diff
@@ -55,10 +55,23 @@ int main(int argc, char **argv)
   QMCGaussianParserBase *parser=0;
   int iargc=0;
   std::string in_file(argv[1]);
+
+  
   std::string punch_file;
   std::string psi_tag("psi0");
   std::string ion_tag("ion0");
-  std::string prefix("sample");
+  std::string prefix(argv[2]);
+
+
+  std::string delimiter =".out";
```

Comment:
I think we could improve the handling here. Would it be simpler to add the requirement that the suffix actually is *.out ? Is this completely standard for GAMESS output?

Reply:
this will pick anything that has .out* (so out or output will be taken)